setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name='fyle',
-    version='v0.2.0',
+    version='v0.3.0',
```

Comment:
Question: How do we decide version number?

Reply:
major.minor.patch

if there is a breaking change then increase major version, else minor for new features etc
patch is used when we fix on a particular version

Reply:
Okay, so since `approver` is a new feature `v0.2.0` -> `v0.3.0` minor change should be right

Reply:
yup